motor/frameworks/tornado/__init__.py

```diff
@@ -83,6 +71,10 @@ def chain_return_value(future, loop, return_value):
     return values resolve a Future with it, and are implemented with callbacks
     rather than a coroutine internally.
     """
+    warnings.warn(
+        "The chain_return_value function is deprecated and will be removed in"
+        "Motor 3.0", DeprecationWarning, stacklevel=2)
```

Comment:
If we're still using this method internally doesn't that mean we can't deprecate it?

Reply:
Correct. I removed **DEPRECATED** from the docstring and developer-guide but forgot to remove the warning.